src/main/webapp/WEB-INF/swagger.yaml

```diff
@@ -60,6 +60,41 @@ paths:
           description: "successful operation"
           schema:
             type: "string"
+  /billpay:
+    post:
+      tags:
+      - "Accounts"
+      summary: "Pay bill"
+      description: ""
+      operationId: "billpay_1"
+      consumes:
+      - "application/json"
+      - "application/xml"
```

Comment:
Let's remove application/xml from our proxy since we are never going to send XML from angularJS.